assets/js/theme/wishlist.js

```diff
@@ -13,6 +13,8 @@ export default class WishList extends PageManager {
         this.options = {
             template: 'account/add-wishlist',
         };
+
+        return this;
```

Comment:
Any particular reason for doing a `return` here ?

Reply:
yeah, its so we can chain a call to one of it's methods:  https://github.com/bigcommerce/cornerstone/pull/1134/files#diff-9c4b0efa294566acaa5d14e862d35bc7R20